content/api/customer-alerts.apib

```diff
@@ -0,0 +1,314 @@
+FORMAT: 1A
```

Comment:
This should probably replace the current `alerts.apib` docs.

Reply:
Done.